xml/obs_best_practice_staging_workflow.xml

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 version="5.1" xml:id="staging_workflow_how_to"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink" >
+  <title>Staging Worlflow</title>
+  <para></para>
+  <sect2>
+    <title>Creating a Staging Workflow</title>
+    <para>
+      At the moment the Staging Workflow is only accessible by hand.
+      "<emphasis>/staging_workflows/new?project=openSUSE:Factory</emphasis>"
+      The creation of a Staging will automatically create two staging projects
+      as a subproject of the main project.
+    </para>
+    <figure>
+     <title>Creating a Staging for openSUSE:Factory</title>
+     <mediaobject>
+      <imageobject>
+       <imagedata fileref="staging_workflow_create.png"/>
+      </imageobject>
+     </mediaobject>
+    </figure>
+    <note>
+      <para>
+        An Admin should previously create the manager groups.
+        Those managers would be the maintainers of the staging projects
+      </para>
+    </note>
+  </sect2>
+  <sect2>
+    <title>Start using the Staging Workflow</title>
+    <para>
+      After creating the Staging, we can start using it. In this view, we could
```

Comment:
There are a few cases where we call it Staging and some others where it's Staging Workflow. We should decide for one them and use it everywhere.

Reply:
I think just calling it ``Staging`` sounds weird. 

``After creating a Staging...`` <- what staging? Idk...

But fine with it for now ...